Automattic/bugomattic
# Add subheaders to each step

#### What Does This PR Add/Change?

This PR adds subheaders to each step! 🎉 

**Type step:** Identify the type of issue you're reporting. The type you select will guide the reporting process.

**Feature selection step:** Explore the tree or use the search function to select a feature. The search results will include exact word matches from descriptions, as well as matches based on related keywords.

**Next Steps step:** Review the tasks listed below, carefully curated by our feature team. We've customized the links to speed up your reporting process.


#### Testing Instructions



* Make sure that no tests are failing. 
* Make sure that the subheaders are displayed in each step. 

#### Issues



Related to #  
Closes #54
